task/catlin-validate/0.1/tests/multiple-input-configmap.yaml

```diff
@@ -0,0 +1,9 @@
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: catlin-input-multiple
+data:
+  input.txt: |
+    task/hello/0.1/hello.yaml
+    task/bye/0.1/bye.yaml
```

Comment:
I have used a configmap since the input to the task is a file in a workspace instead of array params. This (https://github.com/tektoncd/catalog/pull/1178#discussion_r1286214969) is the reason to keep it a file in a workspace rather than input parameter array.